src/components/forms/sections/BerthOverview.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { Col, Container, Row } from 'reactstrap';
+
+import Agreement from '../fragments/Agreement';
+import ApplicationCode from '../fragments/ApplicationCode';
+import Newsletter from '../fragments/Newsletter';
+import BerthOverviewInfo from './BerthOverviewInfo';
+
+import { ApplicationState } from '../../../redux/types';
+import { Berths } from '../../berths/types';
+import { StepType } from '../../steps/step/Step';
+import { WithBoatType } from '../Selects';
+
+type Props = {
+  values?: {
+    boatName: string;
+    boatRegistrationNumber: string;
+    boatType: string;
+    boatModel: string;
+    boatWidth: string;
+    boatLength: string;
+    boatDraught: string;
+    boatWeight: string;
```

Comment:
Is this file renamed or moved from somewhere to here? I remember those, was thinking if we can use GraphQL typedef for this instead of very long long list

Reply:
Yeah, I just renamed it. It's a good point though, I'll replace them with the auto-generated types.